api/endpoint.go

```diff
@@ -277,7 +277,11 @@ func (request AttributeBetweenSurfacesRequest) execute(
 
 func parseGetRequest(ctx *gin.Context, v Normalizable) error {
 	if err := json.Unmarshal([]byte(ctx.Query("query")), v); err != nil {
-		return core.NewInvalidArgument(err.Error())
+		msg := "Your request could not be processed. " +
+			"Please ensure that the supplied data is valid " +
+			"and conforms to the expected swagger specification: "
+		return core.NewInvalidArgument(
```

Comment:
Can we have a test? :pleading_face: 

ErrorHTTPResponse tests in main_test.go are responsible for making sure our errors reach users and don't get lost along the way. Because of the duplication in the server code that we had they are repeated for every endpoint separately, but I am fine if you choose to stop repeating them for every endpoint and chose just one to test this case on.